File: src/sdk-reference/go/1/collection/update-mapping/snippets/update-mapping.go
mapping := json.RawMessage(`{ "dynamic": "false","_meta": { "area": "Panipokhari" }, "properties":{"plate": {"type": "keyword"}}}`)
err := kuzzle.Collection.UpdateMapping("nyc-open-data", "yellow-taxi", mapping, nil)

if err != nil {
  log.Fatal(err)
} else {
  fmt.Println("Success")
}

File: src/sdk/java/1/controllers/collection/update-mapping/snippets/update-mapping.java
try {
    String mapping = "{\"properties\": {\"plate\": {\"type\": \"keyword\"}}}";

    kuzzle.getCollection().updateMapping("nyc-open-data", "yellow-taxi", mapping);

    System.out.println("Success");
} catch (KuzzleException e) {
    System.err.println(e.getMessage());
}
